src/components/organism/SwapSelectTokenModal/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC } from "react";
+import { ReactComponent as DotToken } from "../../../assets/img/dot-token.svg";
+import { useAppContext } from "../../../stateProvider";
+import Modal from "../../atom/Modal";
+
+type TokenProps = {
+  tokenSymbol: string;
+  tokenId: string;
+  decimals: string;
+  tokenBalance: string;
+};
+
+interface SwapSelectTokenModalProps {
+  setSelectedToken: (tokenData: TokenProps) => void;
+  setIsModalOpen: (isOpen: boolean) => void;
+  selectedTokenA: TokenProps;
+  selectedTokenB: TokenProps;
+  isModalOpen: boolean;
```

Comment:
isModalOpen is too long. Could be just "open" cuz you already said that this interface is reserved for modal